src/unit_tests/dataset_tests.cpp

```diff
@@ -450,8 +450,8 @@ BOOST_AUTO_TEST_CASE(get_transform_test)
     auto ld = locked_dataset(uri_options1);
     auto actual = std::vector<double>();
     auto expected = std::vector<double>{
-        -8915910.5905594081, 33.88424960091178, 0,
-        5174836.3438357478, 0, -33.88424960091178}; // Manually verified
+        -8915910.5905594081, 33.88424960091165, 0,
+        5174836.343835746, 0, -33.88424960091165}; // Manually verified
```

Comment:
Angry doubles.

Reply:
Real-valued comparisons should prefer to use a test of the form `abs(target - actual) < EPSILON` for some appropriately set `EPSILON`.

Reply:
yea, I wanted to minimize changes; but agree; reverted these numbers change and added a better comparison: https://github.com/geotrellis/gdal-warp-bindings/pull/120/commits/3f2678ef9f697cad0944386e6ae3891f9d804446